### DISCUSSION TITLE
Synology Scheduled Task fails but same command works ran through SSH

### DISCUSSION THREAD

**Discussion by FezVrasta** [Taxel/PlexTraktSync#1536] [Support] [answered]
Hi, if I run `plextraktsync` from my Synology user (not root) while connected through SSH everything works perfectly. But if I try to run it from a Scheduled Task configured through DSM7 I get the following error:

```
/opt/bin/python3.10: /opt/lib/libc.so.6: version `GLIBC_2.34' not found (required by /usr/lib/openhook.so)
/opt/bin/python3.10: /opt/lib/libc.so.6: version `GLIBC_2.34' not found (required by /usr/lib/libapparmor.so.1)
/opt/bin/python3.10: /opt/lib/libc.so.6: version `GLIBC_2.33' not found (required by /usr/lib/libapparmor.so.1)
```

Could anyone help me debug the issue please? It used to work fine in the past but stopped after a DSM update I think.

**Comment by glensc** [comment, marked as answer]
your `/opt/bin/python3.10` is broken. no idea how you got it to your system. out of scope of this project.

you may want to re-check file permissions of all files, python and the libraries and the directories they belong
